src/onnxruntime.cc

```diff
@@ -795,11 +793,9 @@ ModelInstanceState::ValidateTypedSequenceControl(
       return TRITONSERVER_ErrorNew(
           TRITONSERVER_ERROR_INVALID_ARG,
           (std::string("unable to load model '") + model_state_->Name() +
-           "', sequence control '" + tensor_name +
-           "', the model expects data-type " +
-           OnnxDataTypeName(iit->second.type_) +
-           " but the model configuration specifies data-type " +
-           tensor_datatype)
+           "', configuration expects datatype " + tensor_datatype +
+           " for sequence control '" + tensor_name + "', model provides " +
+           OnnxDataTypeName(iit->second.type_))
```

Comment:
The original error message for this and above already mentioned the config / model difference

Reply:
Agreed, I would keep the above 2 the same as they were

Reply:
Sure, make this change